pkg/metrics/exporters/view/view.go

```diff
@@ -0,0 +1,78 @@
+package view
+
+import (
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func Views() []metric.View {
+	return []metric.View{
+		metric.NewView(
```

Comment:
We should probably keep defining views in the reporters. This could become a repository of views (e.g. this holds a list of views, reporters append to this list by calling AddView())